draft-packages/form/KaizenDraft/Form/SearchField/SearchField.module.scss

```diff
@@ -0,0 +1,6 @@
+@import "~@kaizen/design-tokens/sass/spacing";
+
+.label {
+  display: block;
+  margin-bottom: $spacing-xs; // @note: add gap between label and input
```

Comment:
Could you add `flex` to the wrapping `div` with `gap` instead?

Perhaps an opportunity for a `FieldWrapper` to keep this consistent across all fields

Reply:
Hmm... It might need more investigation for that, because our form fields also tend to have validation messages and descriptions below them (you'll see this in `TextField`). Using `gap` might not be suitable, unless the pattern is that those are outside of the container.

Reply:
Yeah I'd probably leave out the validation messages from this one, but I'll log a ticket for the `FieldWrapper` to clarify the scope.